doc/reference/views.rst

```diff
@@ -411,6 +411,10 @@ calendar view are:
 ``mode``
     Default display mode when loading the calendar.
     Possible attributes are: ``day``, ``week``, ``month``, ``year``
+``scales``
+    comma-separated list of allowed scales for this view.
+    By default, all scales are allowed.
+    For possible scale values to use in this list, see mode.
```

Comment:
Suggestion: 
comma-separated list of scales to provide. By default, all scales are available. See mode for possible scale values.